samples/libcurl/build.gradle

```diff
@@ -8,12 +8,18 @@ plugins {
 group 'org.jetbrains.kotlin.sample.native'
 version '1.0'
 
+def localRepo = Paths.get(System.getProperty('user.home'), '.m2-kotlin-native-samples')
```

Comment:
Why not placing the repository somewhere under `samples` or even `samples/libcurl` directory?

Reply:
Fixed